src/Components/__stories__/PopoverExamples/DisplayPopoverOnBlurExample.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+
+import Input from "../../Input"
+import { OverlayTrigger } from "../../OverlayTrigger"
+import { Popover } from "../../Popover"
+
+export class DisplayPopoverOnBlurExample extends React.Component {
+  state = {
+    value: "",
+    show: false,
+  }
+
+  render() {
+    const { show } = this.state
+    const popover = <Popover>Is your email {this.state.value}?</Popover>
+
+    return (
+      <div style={{ padding: "50px" }}>
```

Comment:
I know it's a small/minor style, but I would probably use styled-components here anyway to stay consistent.

Reply:
Paddings and margins are always tricky to consolidate and make consistent (not in terms of coding style but the actual length of padding or margin), so there would be a lot of one-off components and the entire file could be longer than it should if they all should be componentized. I'll componentize them anyway since It does look like most of the `style`s in the storybook can be consolidated.

Ideally, there should be a consolidated margin/padding definitions (e.g. [basscss-margin](http://basscss.com/#basscss-margin)). This is a wider conversation, but I wanted to leave a note that most of these inline `style`s for padding/margin are actually intentional.

Reply:
This is something we should def discuss as we begin implementing artsy-wide style guidelines